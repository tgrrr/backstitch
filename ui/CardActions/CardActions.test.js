import React from 'react'
import renderer from 'react-test-renderer'
import { CardActions } from '.'

it('CardActions: default', () => {
    const component = renderer.create(
        <CardActions>
            <>
                CardActions
            </>
        </CardActions>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
