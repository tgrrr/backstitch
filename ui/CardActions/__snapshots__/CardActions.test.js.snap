// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardActions: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Cardactions "
    data-testid="Cardactions"
  >
    <div
      className="MuiCardActions-root MuiCardActions-spacing"
    >
      CardActions
    </div>
  </div>
</div>
`;
